package/com.unity.formats.usd/Documentation~/USD Export.md

```diff
@@ -0,0 +1,77 @@
+# Exporting USD files
+
+USD files can be created by exporting GameObjects selected in the Hierarchy using the USD context menu options, or using the Recorder window. Supported USD file formats are:
+
+* USDC (.usd or .usdc)
+* USDA (.usda)
+* USDZ (.usdz)
+
+The default file format when exporting is .usd with USDC encoding, but changing the file ending in the save menu to .usda will export as USDA.
+
+## Export Selected with Children
+
+On Mac you will see multiple file format options- USDC and USDA.
+
+On Windows the default is .usd with a USDC format, but you can change the file ending in the Save window to specify a different file ending and encoding.
+
+
+## Export Transform Overrides
+
+Modifications to transforms in USD files can be exported as overrides to the original files. 
```

Comment:
Sounds like a limitation for 'known limitations', I'm not sure? Is this maybe this is just the more 'general' limitation that Unity and USD have been developed with differing goals I guess.

Reply:
I'm not sure what you mean of this being a limitation? Exporting the Transform Overrides is a different import option to plain export, exporting only the USD 'Overs' for a USD file instead of the full file. It's a USD composition concept to allow layering. Does this answer your question?

Reply:
This is an odd one... this comment was on line 48 ... confused!

> Note: Modifications to the transform of the Root GameObject are not currently reflected in the override, as Unity assumes the root in all USD files is at the origin.

Reply:
Oh! Yes in that case I completely understand what you mean by Unity and USD being developed differently. Yes I guess it is a limitation in that case that the Exporter always sets the root to the origin. This feels a lot more specific than the limitations currently called out in that section, kind of like this is more of an implementation detail? If I was looking for the reason why my overrides were being exported at the root I'd probably look for it in this specific section, rather than a broad limitations section. What do you think?

Reply:
I definitely think you are right. I'm still learning our stack so being a bit nit-picky. I'd be fine with sorting this out in a future revision, as a 'user' isn't necessarily going to read through all the documentation. This could be addressed as FAQ and/or exporter/importer specific issues section at a later date.